test/tests/functional/pbs_dup_acc_log_for_resv.py

```diff
@@ -51,6 +52,52 @@ def setUp(self):
             self.skipTest('test requires two MoMs as input, ' +
                           'use -p moms=<mom1>:<mom2>')
 
+    def check_log_resv_time(self, accounting_log_msg, start_tup,
+                            end_tup, check_status):
+        """
+        This is a helper function to reterive start and end time
+        of standing rervation from accounting log.
+        """
+        find_start = m[start_tup][end_tup].find("start=")
+        find_end = m[start_tup][end_tup].find("end=")
+        find_duration = m[start_tup][end_tup].find("duration=")
+        if check_status == "start":
+            time_str = m[start_tup][end_tup][
+                find_start:find_start + (find_end - find_start - 1)]
+            new_time_str = time_str.replace("\'", "")
+            actual_reservation_time = new_time_str.split("=")[1]
+        elif check_status == "end":
+            time_str = m[start_tup][end_tup][
+                find_end:find_end + (find_duration - find_end - 1)]
+            newstr = time_str.replace("\'", "")
+            actual_reservation_time = newstr.split("=")[1]
+        return int(actual_reservation_time)
+
+    def differentiate_resv_instance(self, expected_start, expected_end,
+                                    got_start, got_end, reserve_index):
+        """
+        This function to differentiate log match
+        for each instance of standing reservation
+        on basis of comparing expected and got start and end time
+        of standing reservation of each instance(reserve_index)
+        """
+
+        self.assertTrue(((expected_start == got_start) and
+                         (expected_end == got_end)),
+                        'Got time not matched with exepected time in'
+                        'accounting log time for standing reservation  ')
```

Comment:
How about 'Time in the accounting log does not match the expected time for the standing reservation' ?